### PR TITLE
Fix/tr 3287/update raphael and eve

### PR DESCRIPTION
**Pull request by krampstudio**
Related to https://oat-sa.atlassian.net/browse/TR-3287

 - update rahpael to 2.3.0 in order to get rid of the subdependency to `eve` using `git://`
 - remove the local copy of raphael :exploding_head: 
 - link the node_modules copy in requires mapping

**How to test**
 - test graphic interaction in debug mode
 - rebundle locally (`cd tao/views/build && npm i && npx grunt bundleall`) and test again the graphic interactions

### CODE REVIEW

File: views/build/config/requirejs.build.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "moment": "../node_modules/moment/min/moment-with-locales",
     "handlebars": "../node_modules/handlebars/dist/handlebars",
     "class": "lib/class",
-    "raphael": "lib/raphael/raphael",
+    "raphael": "../../../tao/views/node_modules/raphael/raphael",
```
Comment (wazelin) [inline views/build/config/requirejs.build.json:21]:
```suggestion
    "raphael": "../node_modules/raphael/raphael",
```

**Review by wazelin — CHANGES_REQUESTED** [review]

**Comment by wazelin** [comment]
Not sure why we have [this version here](https://github.com/oat-sa/tao-core/blob/8f7a6465b7d5691120ce289f7fc44960044deede/views/package.json#L3) in the first place, but should we also update it?

**Comment by krampstudio** [comment]
> Not sure why we have [this version here](https://github.com/oat-sa/tao-core/blob/8f7a6465b7d5691120ce289f7fc44960044deede/views/package.json#L3) in the first place, but should we also update it?
npm requires a version but for publishing, we should rather investigate if we mark the package as [private](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private) we can remove it. 
I guess for another PR...

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Lines became thicker**
image

**Pull request merged**